src/OpenTelemetry.Extensions/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Removes `net7.0` target as .NET 7.0 is out
+  of support. The package keeps `netstandard2.0` target, so it
+  can still be used with .NET7.0 apps.
```

Comment:
Do we even want to mention this as net7 is out of support?

Reply:
Let's remove it as [.NET 7 has reached End of Support back in May 2024](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core), we don't want to give users an impression that we still support it.

Reply:
I agree with this. If we look at OneCollector the CHANGELOG seems to say using the netstandard2.1 target everything will continue to work on .NET7. But then there is this change which makes that kind of a lie:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2038/files#diff-1cdf207633d83f6336dca31769e6747f3093eb056324f0b1193652f8db01c36fL104

The netstandard2.1 target won't have that logic where the .NET7 target did previously.

I think better to just say (something like) "Dropped support for the `net7.0` target because .NET 7 is no longer supported."